### PR TITLE
AAE-1841 - added delete directive test

### PR DESCRIPTION
**Pull request by VitoAlbano**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
https://issues.alfresco.com/jira/browse/AAE-1841

### CODE REVIEW

File: e2e/content-services/directives/delete-directive.e2e.ts [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+            await contentServicesPage.checkToolbarDeleteIsDisabled();
+            await contentListPage.selectRowWithKeyboard(folderA.entry.name);
+            await contentListPage.dataTable.checkRowIsSelected('Display name', folderA.entry.name);
+            await contentServicesPage.checkToolbarDeleteIsDisabled();
+            await contentListPage.selectRowWithKeyboard(fileTxt.entry.name);
+            await contentListPage.dataTable.checkRowIsSelected('Display name', fileTxt.entry.name);
+            await contentServicesPage.checkToolbarDeleteIsDisabled();
+        });
+
+        afterEach(async () => {
```
Comment (eromano) [inline e2e/content-services/directives/delete-directive.e2e.ts]: move it at the beginning as the other test and for style guide

File: e2e/content-services/directives/delete-directive.e2e.ts [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+    });
+
+    beforeAll(async () => {
+        await this.alfrescoJsApi.login(browser.params.testConfig.adf.adminEmail, browser.params.testConfig.adf.adminPassword);
+        await this.alfrescoJsApi.core.peopleApi.addPerson(acsUser);
+        await this.alfrescoJsApi.core.peopleApi.addPerson(secondAcsUser);
+        await this.alfrescoJsApi.login(acsUser.id, acsUser.password);
+    });
+
+    beforeEach(async () => {
```
Comment (adinapitul) [inline e2e/content-services/directives/delete-directive.e2e.ts]: no need to create the baseFolder in beforeEach and delete it in afterEach.
the baseFolder is not changed in any way in the tests, so this should be moved in beforeAll and delete in afterAll for the suite

File: e2e/content-services/directives/delete-directive.e2e.ts [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+        });
+
+    });
+
+    describe('when user does not have `delete` permission', () => {
+
+        let createdSite = null;
+        let fileTxt, filePdf, folderA, folderB;
+
+        beforeEach(async () => {
```
Comment (adinapitul) [inline e2e/content-services/directives/delete-directive.e2e.ts]: creating the site and adding the user as site member needs to be moved in beforeAll, and deleting the site in afterAll

File: lib/testing/src/lib/content-services/actions/permission.actions.ts [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        return this.alfrescoJsApi.nodes.updateNode(nodeToUpdate.entry.id, payload);
+    }
+
+    disableInheritedPermissionsForNode(nodeId: string): Promise<NodeEntry> {
+        const nodeBody = { permissions: { isInheritanceEnabled: false } };
+        return this.alfrescoJsApi.nodes.updateNode(nodeId, nodeBody, { include: ['permissions'] });
+    }
+
+    enableInheritedPermissionsForNode(nodeId: string): Promise<NodeEntry> {
+        const nodeBody = { permissions: { isInheritanceEnabled: false } };
```
Comment (adinapitul) [inline lib/testing/src/lib/content-services/actions/permission.actions.ts]: this should be "true", not "false"

File: e2e/content-services/directives/delete-directive.e2e.ts (hunk 2 above)
Comment (VitoAlbano) [reply e2e/content-services/directives/delete-directive.e2e.ts]: The trick of the base folder is to clean up the environment after each test, so i with a single delete i'm able to delete all the content i've uploaded as the content are different in some cases. That's why i need to create it every time as in the end i'll delete just that one.

Comment (adinapitul) [reply e2e/content-services/directives/delete-directive.e2e.ts]: ah yes, makes sense for this purpose

**Pull request merged**